### PR TITLE
Update catfacts.go

### PR DESCRIPTION
**Pull request by sloria**
These turned up in a Discord server and made some members feel uncomfortable due to the mentions of abuse and non-survival--things you don't really expect when you're rolling for some wholesome cat facts

### CODE REVIEW

**Comment by JordiHakimZiyech** [comment]
i'd suggest making your own "catfact" command then. those might not be your cute facts you want. but other people do want these, as they're facts & give information.

**Comment by sloria** [comment]
Fair enough. Perhaps just a content warning and spoiler text, then? See latest commit.

**Comment by l-zeuch** [comment]
Might be a feasible solution everyone can be happy with.
Those who don't want to see it just don't have to click then, so that's a fair and nice approach!

(Don't PR to `master` though > [CONTRIBUTING.md](https://github.com/jonas747/yagpdb/blob/master/CONTRIBUTING.md))

**Comment by sloria** [comment]
Woops. Changed the base branch.

**Comment by l-zeuch** [comment]
No worries, accidents happen 😄

File: stdcommands/catfact/catfacts.go [hunk 1]
```diff
@@ -114,8 +114,8 @@ var Catfacts = []string{
...
-	`During the time of the Spanish Inquisition, Pope Innocent VIII condemned cats as evil and thousands of cats were burned. Unfortunately, the widespread killing of cats led to an explosion of the rat population, which exacerbated the effects of the Black Death.`,
-	`During the Middle Ages, cats were associated with withcraft, and on St. John’s Day, people all over Europe would stuff them into sacks and toss the cats into bonfires. On holy days, people celebrated by tossing cats from church towers.`,
+	`CW // animal cruelty/death ||During the time of the Spanish Inquisition, Pope Innocent VIII condemned cats as evil and thousands of cats were burned. Unfortunately, the widespread killing of cats led to an explosion of the rat population, which exacerbated the effects of the Black Death.||`,
+	`CW // animal cruelty/death ||During the Middle Ages, cats were associated with withcraft, and on St. John’s Day, people all over Europe would stuff them into sacks and toss the cats into bonfires. On holy days, people celebrated by tossing cats from church towers.||`,
```
Comment (l-zeuch) [inline stdcommands/catfact/catfacts.go:118]:
```suggestion
	`Content warning // animal cruelty/death ||During the time of the Spanish Inquisition, Pope Innocent VIII condemned cats as evil and thousands of cats were burned. Unfortunately, the widespread killing of cats led to an explosion of the rat population, which exacerbated the effects of the Black Death.||`,
	`Content warning // animal cruelty/death ||During the Middle Ages, cats were associated with withcraft, and on St. John’s Day, people all over Europe would stuff them into sacks and toss the cats into bonfires. On holy days, people celebrated by tossing cats from church towers.||`,
```

**Review by l-zeuch — COMMENTED** [review]
For what it's worth, I could imagine not everyone would be familiar with the abbreviation "CW", so expanding that might be a good idea?

File: stdcommands/catfact/catfacts.go (hunk 1 above)
Comment (sloria) [reply stdcommands/catfact/catfacts.go:118]: Sounds good. I learned this abbreviation rather recently myself, so good idea.

Comment (l-zeuch) [reply stdcommands/catfact/catfacts.go]: Prime example 😄
Heard today for the first time about this, as well.

**Pull request merged**